.github/workflows/lint.yml

```
@@ -10,7 +10,7 @@ on:
 jobs:
   golangci:
     name: golangci-lint
-    runs-on: ubuntu-latest
+    runs-on: buildjet-4vcpu-ubuntu-2004
```

```suggestion
    runs-on: ubuntu-latest
```

Same for this one, which is running pretty fast too (cf https://github.com/cosmos/cosmos-sdk/actions/workflows/lint.yml)